pgx/bridge_bidding.py

```diff
@@ -94,11 +95,25 @@ def env_id(self) -> v1.EnvId:
 
 
 class BridgeBidding(v1.Env):
-    def __init__(self):
+    def __init__(self, dds_hash_table=None):
```

Comment:
```suggestion
    def __init__(self, *, dds_hash_table_path: Optional[str] =None):
```

- なにを引数に求められているのかわからない
  - 型ヒントがない
  - 変数名からわからない
- 他の環境にならってパラメータ引数指定を必須にする